### ISSUE TITLE
how to get the Node GPU value using java io.fabric8 kubernetes-client library

### ISSUE THREAD

**Issue by brinst07** [fabric8io/kubernetes-client#4042]
Hello, first of all, thank you for creating a great library.

kubernetesClient.top().nodes().metrics().getItems()

If you use the above command, only cpu and memory are checked for metric.

However, if the gpu is inquired in the assigned node with the describe command, you can see that the information about the gpu is printed as follows.
![image](https://user-images.githubusercontent.com/60083557/161934008-d59b323a-acec-4174-9fba-907951579813.png)

Is there any way to get the value for gpu?

**Comment by manusa** [comment]
You're requesting metrics when you should be actually getting the node information:
```
kc.nodes().withName("your-node").get();
```

That should provide you with the same data as the kubectl get nodes describe command. Which is what your screenshot is showing, right?

**Comment by brinst07** [comment]
I entered the following command:
kubectl describe node <node-name>

**Comment by manusa** [comment]
> I entered the following command: kubectl describe node

That's a combination of:
```
GET /api/v1/nodes/your-node
GET /api/v1/pods?fieldSelector=spec.nodeName%3Dyour-node%2Cstatus.phase%21%3DFailed%2Cstatus.phase%21%3DSucceeded
GET /api/v1/events?fieldSelector=involvedObject.name%your-node%2CinvolvedObject.namespace%3D%2CinvolvedObject.kind%3DNode%2CinvolvedObject.uid%3Dyour-node
GET /apis/coordination.k8s.io/v1/namespaces/kube-node-lease/leases/your-node
```

So you need to perform those calls from the client, and combine the necessary data:
```java
kc.nodes().withName("your-node").get();
client.pods().withField("spec.nodeName", "your-node").list(); // Then filter Failed and Succeeded
client.v1().events()
  .withField("involvedObject.name", "your-node")
  .withField("involvedObject.kind", "Node")
  .list();
client.leases().inNamespace("kube-node-lease").withName("your-node").get();
```

Basically you get the specs from the Node information, and compute yourself the usage with the Node's Pods information.

This is more or less what I'm doing in this screen in YAKC's dashboard:

![image](https://user-images.githubusercontent.com/488391/165445606-81d1bea5-721c-4e7c-a753-ee8fba4a7ef9.png)

TL;DR info is not from metrics, but [computed](https://github.com/manusa/yakc/blob/257ceccfd0e7fb5b62d22ec259f8e71e3834ec80/quickstarts/quarkus-dashboard/src/main/frontend/src/nodes/NodesDetailPage.js#L44) from Node spec and Pod status

**Comment by brinst07** [comment]
Thank you for your response.
The current gpu resource is output because the gpu operator is installed on the node.

[https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/getting-started.html](gpuOperator)

Do you have a plan to output gpu allocated resource information by supporting the plugin?
